### PR TITLE
[bitnami/*] Readme automation

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**

This PR  adds a comment to the beginning of each file so our internal automation is able to know the name used for the application in the current chart version.
Also, it standardizes the READMEs headers.

All those changes allow us to maintain internally a single source of truth for trademarks, descriptions, names, etc that will be automatically updated.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by marcosbc** [comment]
Could you add more context + an explanation of what this achieves?

**Comment by miguelaeh** [comment]
Hi @marcosbc ,
I am doing some internal testing for a new feature of the pipeline, that's why the PR is marked as a draft. I will add the proper information once everything is properly working and I complete the PR for all the charts

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/phpmyadmin/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# phpMyAdmin
+<!--- app-name: phpMyAdmin -->
 
-[phpMyAdmin](https://www.phpmyadmin.net/) is a free and open source administration tool for MySQL and MariaDB. As a portable web application written primarily in PHP, it has become one of the most popular MySQL administration tools, especially for web hosting services.
+# phpMyAdmin
 
+[phpMyAdmin](https://www.phpmyadmin.net/) is a free and open source administration tool for MySQL and MariaDB. 
 ## TL;DR
```
Comment (carrodher) [inline bitnami/phpmyadmin/README.md:6]:
```suggestion
[phpMyAdmin](https://www.phpmyadmin.net/) is a free and open source administration tool for MySQL and MariaDB. 

## TL;DR
```

File: bitnami/common/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Bitnami Common Library Chart
+<!--- app-name:  -->
```
Comment (carrodher) [inline bitnami/common/README.md:1]: Take a look at this one, common seems to be special
